src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    method="POST"
+    data-qa="big"
+    class="form"
+    >
```

Comment:
Keep an eye on indentations.

```suggestion
    <form
      method="POST"
      data-qa="big"
      class="form"
    >
```